MapleServer2/Packets/CinematicPacket.cs

```diff
@@ -60,12 +60,12 @@ public static PacketWriter StartSceneSkip()
         return pWriter;
     }
 
-    public static PacketWriter Conversation(int npcId, string stringId, int delay, Align align)
+    public static PacketWriter Conversation(int npcId, string illustId, string stringId, int delay, Align align)
```

Comment:
Minor but does `illust` mean `illustration`? If so, probably better to name it that, if not then nevermind

Reply:
It is, i just copied from the trigger and didn't think about it. Will fix tomorrow and merge